### DISCUSSION TITLE
Error 403

### DISCUSSION THREAD

**Discussion by JMOJEDA123** [community/community#123561] [New to GitHub]
i have this error: 
remote: Permission to JMOJEDA123/PreEntrega1_JoseOjeda.git denied to omec2323.
fatal: unable to access 'https://github.com/JMOJEDA123/PreEntrega1_JoseOjeda.git/': The requested URL returned error: 403

how can i solve this?

**Comment by davevad93** [comment]
Hi @JMOJEDA123 ,

This error usually means that there is a problem with your Git credentials, first make sure you have correctly set up your user email, which has to be the same as your GitHub account by entering this command:

- `git config --list`

If your email doesn't match you can configure it with this command:

- `git config --global user.email "youremail@example.com"` 

If the above solutions don't work for you, try creating an SSH Key and use that instead of HTTPS, here's a guide on how to do it:

- [Generating a new SSH key and adding it to the ssh-agent](https://docs.github.com/es/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
- [Adding a new SSH key to your GitHub account](https://docs.github.com/es/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)

If none of the above solutions work, take a look at the answers in this thread:

- [Git push The requested URL returned error: 403](https://es.stackoverflow.com/questions/201496/git-push-the-requested-url-returned-error-403)
